tests/e2e/Jenkinsfile

```diff
@@ -15,9 +15,27 @@ pipeline {
     environment {
         pipeline_report_dir = "pipeline_report2TG"
         tgAuthKey = credentials('TG_authkey_test_report_upload')
+        PRODUCT_NAME = "testguide_report_generator_python"
+        TEST_LEVEL = "system"
     }
 
     stages {
+        stage('Set Product Version') {
+            steps {
+                script {
+                    pyprojectContent = readFile 'pyproject.toml'
+                    // This pattern matches versions like "1.0", "1.1.0", "1.1.0-beta", etc.
+                    pattern = ~'version\\s*=\\s*"(\\d+(\\.\\d+)*(-\\w+)?"'
```

Comment:
There is a closing parenthesis missing at the end. Also the it would allow a version like "1"

Idea: 
`version\\s*=\\s*"(\\d+(\\.\\d+){1,2}(-\\w+)?)"`

Reply:
sorry wrong account xD